tests/unit/DetailComment.spec.ts

```diff
@@ -42,49 +42,38 @@ describe('DetailComment', () => {
     expect(vm.value).toBe('Initial comment')
     expect(vm.placeholder).toBe('Enter Comment Here')
     expect(vm.autofocus).toBe(true)
-
-    wrapper.destroy()
   })
 
-  // FUTURE: fix this test
-  it.skip('emits valid event when value prop is changed', async () => {
-    const wrapper = shallowMount(DetailComment,
+  it('emits valid event when value prop is changed', async () => {
+    const wrapper = mount(DetailComment,
       {
+        props: {
+          value: 'testing 1 2 3'
```

Comment:
This isn't the same as the previous test.

Previously, we started with no props and then set them. Now, you start with initial props.

I'd prefer to see the previous unit test left alone, skipped if you can't get it working. Add a new test if needed.

Also, why did you remove old line 65?

Reply:
For this unit test, earlier there was a line `wrapper.setProps({ value: 'testing 1 2 3' })` which I just changed to setting prop while mounting.  I will revert it back and skip the test.

Reply:
Sure.

"setProps" is different from mounting the component with props. In the former, you are _changing_ the props. In the latter, they are initialized with those values. The former can cause getters and watchers to react.

Reply:
Thanks for the new test (in addition to the original, skipped test).

FWIW, I consider this conversation resolved.